packages/web/app/api/metrics/route.ts

```
@@ -18,8 +18,8 @@ const metrics = {
   startTime: Date.now(),
 }
 
-// Export function to update metrics from other parts of the app
-export function updateMetric(metric: keyof typeof metrics, value: number = 1) {
+// Function to update metrics (internal use only)
+function updateMetric(metric: keyof typeof metrics, value: number = 1) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)